docs/markdown/snippets/test_args.md

```diff
@@ -0,0 +1,13 @@
+## `test()` and `benchmark()` args kwarg replaced by positional arguments
+
+For consistency with other functions, `test()` and `benchmark()` now takes
+command argument together with the executable as positional arguments.
+`test('something', exe, args: ['--foo'])` now becomes simply
+`test('something', exe, '--foo')`.
+
+In addition, more types are accepted both as executable and as arguments, to be
+consistency with what `custom_target()`'s command keyword argument supports:
+strings, build target, custom targets, index of custom targets, external programs
+and files.
+
+Similar change applies to `rust.test()` method.
```

Comment:
Why not just list all 3 at the same time?

Reply:
Indeed, fixed.